spec/lib/inclusion_intersection_validator_spec.rb

```diff
@@ -24,7 +24,7 @@ class BadSetupValidator
   class ListValidator
     include ActiveModel::Validations
     attr_accessor :list
-    validates_with ActiveModelValidatesIntersectionOf::Validator, attributes: [:list], in: VALID_ARRAY, message: "not valid"
+    validates_with ActiveModelValidatesIntersectionOf::Validator, attributes: [:list], within: VALID_ARRAY, message: "not valid"
```

Comment:
Can you duplicate this class, so we can test separated a list with the option `in:` and another list using `within:`
(you can create a context and just change the subject class inside each describe, considering that  you will use one describe for each scenerario)